app/views/layouts/application.html.erb

```diff
@@ -12,6 +12,11 @@
 
   <body>
   <div><h1>Bloggy App</h1></div>
+  <div class="sign-put">
+    <%= link_to "Sign out", destroy_user_session_path, method: :delete %>
```

Comment:
- [ ] If you use `button_to` instead of the link, the sign-out will work without giving you an error.
>It will be something like the following:
```ruby
 <%= button_to "Sign out", destroy_user_session_path, method: :delete %>
```